.rubocop.yml

```diff
@@ -21,10 +21,6 @@ Metrics/BlockLength:
   Exclude:
     - 'spec/**/*'
 
-Metrics/BlockLength:
-  Exclude:
-    - spec/**/*.rb
```

Comment:
why did this get removed? It's quite usual for rspec blocks to get longer than 5 lines.

Reply:
Repeat of rule above.